src/dc_tools.rs

```diff
@@ -296,6 +298,16 @@ pub(crate) async fn dc_delete_file(context: &Context, path: impl AsRef<Path>) ->
     }
 }
 
+pub async fn dc_delete_files_in_dir(context: &Context, path: impl AsRef<Path>) {
+    if let Ok(mut read_dir) = async_std::fs::read_dir(path).await {
```

Comment:
this should probably log at least a warning if it is not able to read the directory

Reply:
Done